superset-frontend/src/components/Menu/Menu.less

```diff
@@ -39,3 +43,42 @@
     }
   }
 }
+
+#secondary-menu {
+  margin-top: -20px;
+  .navbar-header .navbar-brand {
+    font-weight: @font-weight-bold;
+  }
+  .navbar-right {
+    .btn-default {
+      text-transform: uppercase;
+      font-weight: 500;
+      font-size: @font-size-s;
+      background-color: @brand-primary;
+      border-radius: 4px;
+      color: @lightest;
+      margin: 8px 43px;
+      padding: 8px 51px 8px 43px;
+      border: none;
+      i {
+        padding: 4px @font-size-xs;
+      }
+    }
+  }
+  .navbar-nav {
+    li {
+      a {
+        font-size: @font-size-s;
+        padding: 8px;
+        margin: 8px;
+        color: @gray-drak-heading;
+      }
+    }
+    li.active > a,
+    li > a:hover {
+      background-color: @gray-light-hover;
+      border-bottom: none;
+      border-radius: 4px;
+    }
+  }
+}
```

Comment:
Hmm, should we move these to styled components or are we ok with living in a world with a mix of less + css-in-js?

Reply:
@ktmud good suggestion. I think moving to these to styled components will be cleaner. I  will move css to the component.